opus/application/static_media/js/browse.js

```diff
@@ -471,6 +471,7 @@ var o_browse = {
             },
             stop: function(event, ui) {
                 o_browse.onUpdateSlider(ui.value);
+                $("#galleryView").modal('hide');
```

Comment:
Now that you're killing the modal when the cache empties, I'm not sure this is necessary. As long as the slider is just repositioning the scrollbar with the existing cached data, it's no different than just moving the main scrollbar. We only need to kill the modal when the cache for the viewed observation is deleted.

Reply:
This is not resolved.